spring-kafka/src/main/java/org/springframework/kafka/listener/FailedRecordTracker.java

```diff
@@ -53,6 +53,8 @@
 
 	private boolean resetStateOnRecoveryFailure = true;
 
+	private boolean resetStateOnExceptionChange;
```

Comment:
I'm sorry, do I miss anything?
Looks like this property is out of use...

Reply:
Oops